com.obsproject.Studio.Plugin.obs_kinect.metainfo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.obs_kinect</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>obs-kinect</name>
+  <summary>OBS Plugin to access Kinect data (and setup a virtual green screen based on depth)</summary>
```

Comment:
The summary is used by apps stores, usually in a limited space in the OBS Studio page. So users have context already, the summary doesn't need to mention "OBS Plugin" :slightly_smiling_face:

I suggest shortening this, but I'm not sure how, exactly. I've been asking plugin developers to answer "what can I achieve with your plugin?" in the summary. What comes to my mind is: `Remove background using Xbox Kinect`. Please forgive me if I'm mistaken! I hope I did not misunderstand the purpose of the plugin, and I'm sure you'll be able to come up with a better summary than me.